common/azure-common-test/src/main/java/com/azure/common/test/utils/TestConstant.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.test.utils;
+
+/**
+ * Constant for test classes.
+ */
+public final class TestConstant {
+    public static final int DEFAULT_BUFFER_LENGTH = 1024;
+}
```

Comment:
I see no need for this class - the value could be a private static field inside SdkContext

Reply:
The new commits will remove the class as the Constant is only used at one place.